docs/content/getting-started.md

```diff
@@ -0,0 +1,260 @@
+
+With the toolchain configured, it's time to set up a new Bolt project. This contains the code that handles logic for your app.
+
+If you don’t already have a project, let’s create a new one!
+
+<Tabs groupId="cli-or-terminal">
+<TabItem value="cli" label="Slack CLI">
+
+A blank template can be used to start with project scaffolding:
+
+```sh
+$ slack create first-bolt-app --template slack-samples/bolt-js-getting-started-app
+$ cd first-bolt-app
+```
+
+After a project is created you'll have a `package.json` file with project details and a `.slack` directory for application use.
+
+A few other files exist too, but we'll visit these later.
+
+</TabItem>
+<TabItem value="terminal" label="Terminal">
+
+A blank template can be cloned to start with project scaffolding:
+
+```sh
+$ git clone https://github.com/slack-samples/bolt-js-getting-started-app first-bolt-app
+$ cd first-bolt-app
+$ npm install
+```
+
+Outlines of a project are taking shape, so we can move on to creating an app!
+
+</TabItem>
+</Tabs>
+
+## Running the app {#running-the-app}
+
+Before you can start developing with Bolt, you will want a running Slack app.
+
+<Tabs groupId="cli-or-terminal">
+<TabItem value="cli" label="Slack CLI">
+
+The getting started app template contains a `manifest.json` file with details about an app that we will use to get started. Use the following command and select "Create a new app" to install the app to the team of choice:
+
+```sh
+$ slack run
+...
+[INFO]  bolt-app ⚡️ Bolt app is running!
+```
+
+With the app running, you can test it out with the following steps:
+
+1. Open a direct message with your app or invite the bot to a public channel.
+2. Send "hello" to the current conversation and wait for a response.
+3. Click the attached button labelled "Click me" to post another reply.
+
+After confirming the app responds, celebrate, then interrupt the process by pressing `CTRL+C` in the terminal to stop your app from running.
+
+</TabItem>
+<TabItem value="terminal" label="Browser">
+
+Navigate to your list of apps and [create a new Slack app](https://api.slack.com/apps/new) using the "from a manifest" option:
+
+1. Select the workspace to develop your app in.
+2. Use copied values from the `manifest.json` file to create your app.
+3. Confirm the app features and click "Create".
+
+You'll then land on your app's **Basic Information** page, which is an overview of your app and which contains important credentials:
+
+![Basic Information page](/img//basic-information-page.png "Basic Information page")
+
+To listen for events happening in Slack (such as a new posted message) without opening a port or exposing an endpoint, we will use [Socket Mode](/concepts/socket-mode). This connection requires a specific app token:
+
+1. On the **Basic Information** page, scroll to the "App-Level Tokens" section and click **Generate Token and Scopes**.
+2. Name the token "Development" or something similar and add the `connections:write` scope, then click "Generate".
```

Comment:
suggestion: In the section below, I think we bold the sections a well. This makes it a little easier to read at a glance.:
```suggestion
1. On the **Basic Information** page, scroll to the **App-Level Tokens** section and click **Generate Token and Scopes**.
2. Name the token "Development" or something similar and add the `connections:write` scope, then click **Generate**.
```